01_fib/jonathanheaden/main_test.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestMainOutput(t *testing.T) {
+	// Given
+	r := require.New(t)
+	var buf bytes.Buffer
+	out = &buf
+
+	// When
+	main()
+
+	// Then
+	expected := strconv.Quote("\n1\n1\n2\n3\n5\n8\n13\n\n-1\n-1\n-2\n-3\n-5\n-8\n-13\n")
```

Comment:
The output of the negative fib series is not correct. Double-check the wikipedia page. The sign alternates.

Reply:
I had to think through this a lot, I see what you mean. I have altered the test and code